GSA/digitalgov.gov
# fix search support email

## Summary

A potential search.gov user has been attempting to reach out to us via the old search.gov support email that he found on digital.gov.  This fixes the search.gov support email across the digital.gov footprint.

## Preview

[Link to Preview]()



## Solution

I changed search@support.digitalgov.gov to search@gsa.gov




## How To Test

Confirm that the search.go support email has been changed from search@support.digitalgov.gov to search@gsa.gov
